homeassistant/components/doorbird/__init__.py

```diff
@@ -126,50 +115,43 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         )
         raise ConfigEntryNotReady
 
-    token: str = doorstation_config.get(CONF_TOKEN, config_entry_id)
-    custom_url: str | None = doorstation_config.get(CONF_CUSTOM_URL)
-    name: str | None = doorstation_config.get(CONF_NAME)
-    events = doorstation_options.get(CONF_EVENTS, [])
-    doorstation = ConfiguredDoorBird(device, name, custom_url, token)
-    doorstation.update_events(events)
+    token: str = door_station_config.get(CONF_TOKEN, config_entry_id)
+    custom_url: str | None = door_station_config.get(CONF_CUSTOM_URL)
+    name: str | None = door_station_config.get(CONF_NAME)
+    events = door_station_config.get(CONF_EVENTS, [])
```

Comment:
@bdraco this was using options flow data previously best I can tell.

Reply:
indeed it was. good catch

Reply:
fixed in e60fca009765b3cdabb55839d8fe62fff32509c0

Reply:
thanks for catching that 👍

Reply:
Didn't notice it in testing since the events were already registered. Would have had to factory reset the device to notice

Reply:
ran through it line by line just to be sure, but everything else looks good.

Will sleep on it and check again in the morning before merging